src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/HarmonicSchwarzschild.cpp

```diff
@@ -0,0 +1,610 @@
+    const IntermediateComputer<DataType, Frame>& computer,
+    ::Tags::dt<gr::Tags::Shift<3, Frame, DataType>> /*meta*/) {
+  const auto& r = get(get_var(computer, internal_tags::r<DataType>{}));
+  return make_with_value<tnsr::I<DataType, 3, Frame>>(r, 0.);
+}
+
+template <typename DataType, typename Frame>
+Scalar<DataType>
+HarmonicSchwarzschild::IntermediateVars<DataType, Frame>::get_var(
+    const IntermediateComputer<DataType, Frame>& computer,
+    gr::Tags::SqrtDetSpatialMetric<DataType> /*meta*/) {
+  const auto& det_spatial_metric =
+      get_var(computer, gr::Tags::DetSpatialMetric<DataType>{});
+  return Scalar<DataType>(sqrt(get(det_spatial_metric)));
+}
+
+template <typename DataType, typename Frame>
+tnsr::II<DataType, 3, Frame>
+HarmonicSchwarzschild::IntermediateVars<DataType, Frame>::get_var(
+    const IntermediateComputer<DataType, Frame>& computer,
+    gr::Tags::InverseSpatialMetric<3, Frame, DataType> /*meta*/) {
+  const auto& spatial_metric =
+      get_var(computer, gr::Tags::SpatialMetric<3, Frame, DataType>{});
+  const DataType& spatial_metric_00 = get<0, 0>(spatial_metric);
+  const DataType& spatial_metric_01 = get<0, 1>(spatial_metric);
+  const DataType& spatial_metric_02 = get<0, 2>(spatial_metric);
+  const DataType& spatial_metric_11 = get<1, 1>(spatial_metric);
+  const DataType& spatial_metric_12 = get<1, 2>(spatial_metric);
+  const DataType& spatial_metric_22 = get<2, 2>(spatial_metric);
+  const auto& one_over_det_spatial_metric =
+      get_var(computer, internal_tags::one_over_det_spatial_metric<DataType>{});
+
+  tnsr::II<DataType, 3, Frame> inverse_spatial_metric{};
+  get<0, 0>(inverse_spatial_metric) =
+      (spatial_metric_11 * spatial_metric_22 - square(spatial_metric_12)) *
+      get(one_over_det_spatial_metric);
+  get<0, 1>(inverse_spatial_metric) = (spatial_metric_12 * spatial_metric_02 -
+                                       spatial_metric_22 * spatial_metric_01) *
+                                      get(one_over_det_spatial_metric);
+  get<0, 2>(inverse_spatial_metric) = (spatial_metric_01 * spatial_metric_12 -
+                                       spatial_metric_02 * spatial_metric_11) *
+                                      get(one_over_det_spatial_metric);
+  get<1, 1>(inverse_spatial_metric) = (spatial_metric_22 * spatial_metric_00 -
+                                       spatial_metric_02 * spatial_metric_02) *
+                                      get(one_over_det_spatial_metric);
+  get<1, 2>(inverse_spatial_metric) = (spatial_metric_02 * spatial_metric_01 -
+                                       spatial_metric_00 * spatial_metric_12) *
+                                      get(one_over_det_spatial_metric);
+  get<2, 2>(inverse_spatial_metric) = (spatial_metric_00 * spatial_metric_11 -
+                                       spatial_metric_01 * spatial_metric_01) *
+                                      get(one_over_det_spatial_metric);
+
```

Comment:
I checked that all of the equations implemented here are correct in the sense that they agree with my understanding of what the SpEC code does and with the documentation given here, except I never worked out the inverse metric explicitly. Maybe add a quick test to the unit test that confirms metric * inverse metric == identity?